src/analysis/retail/shaman/enhancement/modules/resourcetracker/MaelstromWeaponTracker.tsx

```diff
@@ -20,7 +35,7 @@ const MaelstromWeaponResource: Resource = {
   url: 'maelstrom_weapon',
 };
 
-class MaelstromWeaponTracker extends ResourceTracker {
+export default class extends ResourceTracker {
```

Comment:
minor style thing: setting the class name here can help in debugging to make it clearer what type an object is. not required to merge, though

Reply:
I'll keep this in mind for next time